src/atc_api/main.go

```diff
@@ -61,6 +68,8 @@ func ParseArgs() Arguments {
 	db_driver := flag.String("D", "sqlite3", "database driver")
 	db_connstr := flag.String("Q", "atc_api.db", "database driver connection parameters")
 	warn_only := flag.Bool("W", false, "only warn if the thrift server isn't reachable")
+	ipv4 := flag.String("4", "", "IPv4 address for the API")
```

Comment:
I would rather see an ipv4/v6 domain than an IP. if the given domain is an ip, it will just work.